package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
+    "scss": "node-sass --watch src/stylesheets - dist/stylesheets"
```

Comment:
This command is wrong. We should remove and not fiddle with this.

If you have sass installed globally you could run sass --watch src/stylesheets:dist/stylesheets